File: android/app/src/main/kotlin/dev/tcode/thinmpf/extension/ArtistModelExtension.kt
package dev.tcode.thinmpf.extension

import dev.tcode.thinmpf.model.contract.ArtistModelContract
import dev.tcode.thinmpf.pigeon_output.ArtistDTO

fun ArtistModelContract.toPigeon(): ArtistDTO {
    return ArtistDTO(
        id = this.id.raw,
        name = this.name,
    )
}

File: ios/Runner/Extension/ArtistModelExtension.swift
//
//  ArtistModelExtension.swift
//  Runner
//
//  Created by t k on 2024/08/14.
//

extension ArtistModelContract {
    func toPigeon() -> ArtistDTO {
        return ArtistDTO(
            id: String(self.id.raw),
            name: self.name
        )
    }
}
